File: mesh_inputs.py
def mesh_inputs():
    V0 = 0
    R = 0
    Mag = 0
    RadD = 0

    h = float(input('Height of rectangle ? '))
    l = float(input('Length of rectangle ? '))
    m = int(input('Increments along height ? '))
    n = int(input('Increments along length ? '))
    option = int(input('0:Plane strain or 1:Plane stress ? '))
    E = int(input('Elasticity modulus ? '))
    nu = float(input('Poisson\'s ratio ? '))
    t = float(input('Element thickness ? '))
    print('0: Parabolic shear stress or')
    print('1: Linear bending stress at beam tip or')
    print('2: Beam forced into radius or')
    print('3: Beam transformed to 90 degree section loaded with pressure')
    load_opt = int(input('4: Beam transformed to 90 degree section, inside surface displaced '))

    if load_opt == 0:
        V0 = float(input('Magnitude of total shear force at beam tip ? '))
    elif load_opt == 1:
        V0 = float(input('Magnitude of bending stress at top surface at beam tip ? '))
    elif load_opt == 2:
        R = float(input('Enter radius that beam is deformed into. '))
    elif load_opt == 3:
        R = 2. * l / math.pi
        Mag = float(input('Internal pressure (in undeformed configuration) ? '))
    elif load_opt == 4:
        R = 2 * l / math.pi
        RadD = float(input('Radial displacement ? '))
    R = 0

    filen = raw_input('Write output to which file ? ')
    return h, l, m, n, option, E, nu, t, load_opt, V0, R, Mag, RadD, filen

File: mesh_inputs.m
function [h,l,m,n,option,E,nu,t,loadopt,V0,R,Mag,RadD,filen] = mesh_inputs 

h      = input('Height of rectangle ? ');
l      = input('Length of rectangle ? ');
m      = input('Increments along height ? ');
n      = input('Increments along length ? ');
option = input('0:Plane strain or 1:Plane stress ? ');
E      = input('Elasticity modulus ? ');
nu     = input('Poisson''s ratio ? ');
t      = input('Element thickness ? ');
disp('0: Parabolic shear stress or ');
disp('1: Linear bending stress at beam tip or');
disp('2: Beam forced into radius or');
disp('3: Beam transformed to 90 degree section loaded with pressure');
load_opt = input('4: Beam transformed to 90 degree section, inside surface displaced');
if load_opt == 0
    V0 = input('Magnitude of total shear force at beam tip ? ');
elseif load_opt == 1
    V0 = input('Magnitude of bending stress at top surface at beam tip ? ');
elseif load_opt==2
    R  = input('Enter radius that beam is deformed into. ');
elseif load_opt==3
    R = 2*l/pi;
    Mag = input('Internal pressure (in undeformed configuration) ? '); 
elseif load_opt==4
    R = 2*l/pi;
    RadD = input('Radial displacement ? ');
end
filen  = input('Write output to which file ? ','s');